Services/FeedSyncService.php

```diff
@@ -375,4 +376,51 @@ protected function showProgressEstimateDuration($time_started, $processed, $tota
                  " minutes. Speed: " . $processed_min . " terenas / minute.\n";
         }
     }
+
+    /**
+     *
+     */
+    public function syncRepos($output, $optWall, $show_bar)
+    {
+
+        $dataFolder = $this->dataFolder->locateResource('@PumukitGeantWebTVBundle/Resources/data/repos_data');
```

Comment:
We thought about searching the repo data on the github site, not in the source code, innit?